syllabus/7-Polkadot/13-polkadot-ecosystem-economy-slides.md

```diff
@@ -0,0 +1,652 @@
+
+# Economy
+
+---
+
+## The DOT Token
+
+<div class="grid grid-cols-2">
+
+<div>
+
+<image src="../../assets/img/7-Polkadot/eco/token.avif" style="width: 400px;">
+
+</div>
+
+<div class="text-left">
+
+The DOT token can be in one of the following states:
+
+1. Transferable
+2. Locked (Frozen)
+3. Reserved (Held)
+
+</div>
+
+</div>
+
+---
+
+## Reserved vs Locked Balance
+
+- New terms "Frozen" and "Held" are not quite used in Polkadot yet...
+- DOT in both states belong to the user... but cannot be spent / transferred.
+- Reserved balances stack on top of one another.
+  - Useful for user deposits, or other use cases where there is sybil concerns.
+  - Ex: Deposit for storing data on-chain,
+- Locked balances can overlap each other.
+  - Useful when you want to use the same tokens for multiple use cases.
+  - Ex: Using the same tokens for both staking and voting in governance.
+
+---
+
+## Storage Bloat
+
+One blockchain scaling problem is storage bloat over time.
+
+<br />
+
+Consider the "cost" of storing data on Ethereum:
+
+- A one time gas fee based on the amount of data stored.
+- Once is it placed on the network, it lives there forever, with no additional costs.
+- Over a long enough period of time, the cost of storage per time will reduce to zero.
+
+---
+
+## Storage Deposits
+
+To solve this problem, Polkadot additionally takes a storage deposit (in the form of Reserved Balance) for any data stored in the blockchain.
+
+- This deposit is returned to the user when the user removes the data from the chain.
+
+- This deposit can be quite extreme, since it is returned to the user, and can represent the impermanence or lack of "importance" of the data.
+
+---
+
+## Dust Accounts & Existential Deposit
+
+The most bloat-ful storage on most blockchains are user accounts:
+
+- Both Ethereum and Bitcoin are riddled with "dust accounts" which have such a small balance, they are not worth "cleaning up".
+
+- Polkadot solves this by having an "existential deposit" that all users must hold a minimum amount of DOT, else their account data will be cleaned up.
+
+- Existential deposit can be thought of as a storage deposit for account data.
+
+---
+
+## DOT is a Utility Token
+
+<div class="grid grid-cols-3">
+
+<div>
+
+<image src="../../assets/img/7-Polkadot/eco/utility.avif" style="width: 500px;">
+
+</div>
+
+<div class="text-left col-span-2">
+
+The DOT token serves multiple purposes to help the Polkadot network function:
+
+- Staking
+- Bonding for Parachain Slots / Execution Cores
+- On-Chain Decision Making
+- Value Bearing for Trading / Using
+
+</div>
+</div>
+
+---
+
+## Ideal Usage of DOT Tokens
+
+<image src="../../assets/img/7-Polkadot/ideal-token-distribution.svg" style="width: 1000px;">
+
+Approximately...
+
+Model to be updated after Agile Coretime is live, as parachain slot auctions will be obsolete
+and there are proposals to burn coretime revenue.
+
+Notes:
+
+- 50% Staking / Governance
+- 30% Parachains
+- 20% Tradable / Useable
```

Comment:
Coretime bump @jonasW3F What will be the planned allocations with coretime live?

Reply:
mention burn